tests/storage/test_user_directory.py

```diff
@@ -21,6 +21,7 @@
 ALICE = "@alice:a"
 BOB = "@bob:b"
 BOBBY = "@bobby:a"
+BELA = "@somenickname:a"
```

Comment:
Might be worth putting a comment above this one saying that the localpart is purposefully not `bela` so we can test against display names?